extensions/2.0/Vendor/EXT_feature_metadata/1.0.0/README.md

```diff
@@ -0,0 +1,1665 @@
+`occurrences`|Number of enum occurrences|Enums or fixed-length arrays of enums
+
+Model authors may define their own additional statistics, like `mode` below.
+
+```jsonc
+{
+  "extensions": {
+    "3DTILES_metadata": {
+      "schema": {
+        "classes": {
+          "building": {
+            "properties": {
+              "height": {
+                "type": "FLOAT32"
+              },
+              "owners": {
+                "type": "ARRAY",
+                "componentType": "STRING"
+              },
+              "buildingType": {
+                "type": "ENUM",
+                "enumType": "buildingType"
+              }
+            }
+          }
+        },
+        "enums": {
+          "buildingType": {
+            "valueType": "UINT8",
+            "values": [
+              {
+                "name": "Residential",
+                "value": 0
+              },
+              {
+                "name": "Commercial",
+                "value": 1
+              },
+              {
+                "name": "Hospital",
+                "value": 2
+              },
+              {
+                "name": "Other",
+                "value": 3
+              }
+            ]
+          }
+        }
+      },
+      "statistics": {
+        "classes": {
+          "building": {
+            "count": 100,
+            "properties": {
+              "height": {
+                "min": 3.9,
+                "max": 341.7,
+                "mode": 5.0
+              },
+              "buildingType": {
+                "occurrences": {
+                  "Residential": 70,
+                  "Commercial": 28,
+                  "Hospital": 2
+                }
+              }
+            }
+          }
+        }
+      }
+    }
+  }
+}
+```
+
+## Examples
+
+_This section is non-normative_
+
+The examples below shows the breadth of possible use cases for this extension. 
+
+Example|Description|Image
+--|--|--
+Triangle mesh|Feature IDs are assigned to each vertex to distinguish components of a building.|![Building Components](figures/building-components.png)
+Per-vertex metadata<img width=700/>|An implicit feature ID is assigned to each vertex. The feature table stores `FLOAT64` accuracy values. |![Per-vertex metadata](figures/per-vertex-metadata.png)
+Per-triangle metadata|An implicit feature ID is assigned to each set of three vertices. The feature table stores `FLOAT64` area values.|![Per-triangle metadata](figures/per-triangle-metadata.png)
+Per-point metadata|An implicit feature ID is assigned to each point. The feature table stores `FLOAT64` , `STRING`, and `ENUM` properties, which are not possible through glTF vertex attribute accessors alone.|![Point features](figures/point-features.png)
+Per-node metadata|Vertices in node 0 and node 1 are assigned different `constant` feature IDs. Because the door has articulations these two nodes can't be batched together.|![Per-node metadata](figures/per-node-metadata.png)
```

Comment:
Articulations is not a glTF concept.  This could use a bit more context to say that the nodes are separate so that they can be referenced separately by glTF animations.